<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget_WarpParam</class>
 <widget class="QWidget" name="Widget_WarpParam">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Skeleton Warping</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="cmbSkeletonWarpAlg"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="chkHumanLike">
     <property name="text">
      <string>Keep the result &quot;Human-Like&quot;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="chkShowControl">
     <property name="text">
      <string>Show Control</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>284</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="btnReset">
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>chkHumanLike</sender>
   <signal>toggled(bool)</signal>
   <receiver>Widget_WarpParam</receiver>
   <slot>slotUpdated()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmbSkeletonWarpAlg</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Widget_WarpParam</receiver>
   <slot>slotUpdated()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkShowControl</sender>
   <signal>stateChanged(int)</signal>
   <receiver>Widget_WarpParam</receiver>
   <slot>slotUpdated()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>updated()</signal>
  <slot>slotUpdated()</slot>
 </slots>
</ui>
